trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -339,20 +469,79 @@ def context_relevance_with_cot_reasons(
         Returns:
             float: A value between 0 and 1. 0 being "not relevant" and 1 being "relevant".
         """
-        system_prompt = prompts.CONTEXT_RELEVANCE_SYSTEM
+
         user_prompt = str.format(
             prompts.CONTEXT_RELEVANCE_USER, question=question, context=context
         )
         user_prompt = user_prompt.replace(
             "RELEVANCE:", prompts.COT_REASONS_TEMPLATE
         )
 
+        if criteria and output_space:
+            ContextRelevance.override_critera_and_output_space(
+                criteria, output_space
+            )
+
         return self.generate_score_and_reasons(
-            system_prompt=system_prompt,
+            system_prompt=ContextRelevance.system_prompt.template,
             user_prompt=user_prompt,
             temperature=temperature
         )
 
+    def context_relevance_verb_confidence(
```

Comment:
Would be good to have both `_verb_confidence` and `_cot_reasons` as feedback parameters instead of split as different functions. I see users wanting to use both at the same time. Also this lets us maintain fewer different methods and have less user paralysis on which method to use

Reply:
will do this is the following PR - where I'll planning to continue / pick up the feedback v2 re-design work

Reply:
Since this is planned to be a short-lived method, should we market as internal (_ prefix) or experimental?